src/lightning_app/core/app.py

```diff
@@ -133,6 +136,9 @@ def __init__(
 
         logger.debug(f"ENV: {os.environ}")
 
+        # update index.html
```

Comment:
Make a comment on why this should be in the constructor (it should only be written once during the app lifecycle).